server/routes/referrals/desiredOutcomesView.ts

```diff
@@ -7,7 +7,7 @@ export default class DesiredOutcomesView {
   get checkboxArgs(): Record<string, unknown> {
     return {
       idPrefix: 'desired-outcomes-ids',
-      name: 'desired-outcomes-ids',
+      name: 'desired-outcomes-ids[]',
```

Comment:
We should update the [controller tests](https://github.com/ministryofjustice/hmpps-interventions-ui/blob/b96b8fccceb9e22c19fa0a5d9e6478141c47fb13/server/routes/referrals/referralsController.test.ts#L572-L606) too, I think.